static/app/views/settings/organizationAuditLog/index.tsx

```diff
@@ -73,6 +16,7 @@ type Props = RouteComponentProps<{orgId: string}, {}> &
 type State = AsyncView['state'] & {
   entryList: AuditLog[] | null;
   entryListPageLinks: string | null;
+  eventTypes: string[];
```

Comment:
```suggestion
  eventTypes: string[] | null;
```

On initial load, `eventTypes` is `null` until it's filled by the API request payload.